.github/workflows/helm-chart-ci.yaml

```diff
@@ -170,7 +170,8 @@ jobs:
 
       - name: Run chart-testing (install)
         run: |
-          ct install --config ct.yaml \
+          helm install spire-crds charts/spire-crds
```

Comment:
Should we define the namespace here?

Reply:
We could. Would have to add a kubectl create namespace here too. I figured for the tests it probably didn't matter too much?

Reply:
Done